apps/throttling.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from calendar import monthrange
+from datetime import datetime
+from influxdb_metrics.loader import log_metric
+
+from rest_framework import throttling
+
+LOG = logging.getLogger('transmission')
+
+
+# pylint: disable=attribute-defined-outside-init, super-init-not-called
```

Comment:
I actually think the pylint rule `super-init-not-called` is kind of dumb and we should disable that globally in the .prospector.yml file instead of locally here.